packages/terra-functional-testing/lib/commands/run-axe.js

```diff
@@ -0,0 +1,37 @@
+/* global browser, axe */
+const injectAxe = require('./inject-axe');
+
+/**
+ * Executes axe on the browser.
+ * @param {Object} overrides - The axe options.
+ * @param {Array} overrides.rules - The rule overrides
+ */
+const runAxe = (overrides = {}) => {
+  const isAxeUnavailable = browser.execute(() => window.axe === undefined);
+
+  // Inject axe-core onto the page if it has not already been initialize.
+  if (isAxeUnavailable) {
+    injectAxe();
+  }
+
+  // Extract the axe options for the Terra service from the global browser object.
+  const [, options = {}] = browser.options.services.find(([service]) => (
+    typeof service === 'function' && service.name === 'TerraService'
+  ));
```

Comment:
This logic is duplicated in inject-axe. Could we move this up before the `isAxeUnavailable` check & pass these options down? We could also deconstruction the axe config from the service options and simplify the rules merge as well
```suggestion
  const { axe } = options;
```

Reply:
Consolidated the logic, these were split out intentionally before, but I don't think injectAxe will/should ever be called from anywhere else except this function:

https://github.com/cerner/terra-toolkit/pull/416/commits/2e4d70c646089a29c41346d30db7336ab1a5f41a